toolkit/scripts/utils.mk

```diff
@@ -91,6 +91,7 @@ endef
 # Invoke the above rule for each tracked variable
 $(foreach var,$(watch_vars),$(eval $(call depend_on_var,$(var))))
 
-# Host's extended ACLs influence the default permissions of the
-# files inside the built RPMs. Disabling them for the build directory.
-$(call shell_real_build_only, setfacl -bnR $(PROJECT_ROOT))
+# Host's ACLs influence the default permissions of the
+# files inside the built RPMs. Disabling them for the repository.
+no_repo_acl:
+	@setfacl -bnR $(PROJECT_ROOT)
```

Comment:
While we are here, can we send this to `/dev/null` or fix the warnings it generates? (actually, this may fail on some systems if it returns an error code for those files).